executor/src/witgen/jit/affine_symbolic_expression.rs

```diff
@@ -83,6 +88,12 @@ impl<T: FieldElement, V> ProcessResult<T, V> {
             complete: true,
         }
     }
+    pub fn summary(&self) -> ProcessSummary {
+        ProcessSummary {
+            complete: self.complete,
+            progress: !self.effects.is_empty(),
```

Comment:
We should be careful here, I have seen non-useful range constraints being generated.